tests/test_engine.py

```diff
@@ -1,5 +1,28 @@
-from termcharts.engine import add_char
-from termcharts.engine import render
+import sys
+import pytest
+sys.path.append('.')
+from src.termcharts.engine import (
```

Comment:
Neat, just i prefer one import on each line (black --sl). 

We'll be adding pre-commit for to fix those issues